test/common/stats/stat_merger_test.cc

```diff
@@ -231,6 +230,19 @@ TEST_F(StatMergerThreadLocalTest, newStatFromParent) {
   EXPECT_FALSE(TestUtility::findGauge(store_, "newgauge2"));
 }
 
+TEST_F(StatMergerThreadLocalTest, retainImportModeAfterMerge) {
```

Comment:
nit: add '//' comment above describing the test, especially since it's a little complicated.

Reply:
done; and I also added a value-check.